homeassistant/components/system_bridge/__init__.py

```diff
@@ -113,26 +113,28 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if hass.services.has_service(DOMAIN, SERVICE_SEND_COMMAND):
         return True
 
+    def valid_device(device: str):
+        """Check device is valid."""
+        device_registry = dr.async_get(hass)
+        device_entry = device_registry.async_get(device)
+        if device_entry is not None:
+            return next(
```

Comment:
`next` can raise `StopIteration` if it doesn't find a matching item, if we don't set a default value. We only want to raise `vol.Invalid` from this function.